backend/controllers/auth-controller.js

```diff
@@ -0,0 +1,84 @@
+import { HttpError } from "../helpers/HttpError.js";
+import { prisma } from "../lib/db.js";
+
+const register = async (req, res, next) => {
+  try {
+    const { username, password } = req.body;
+    const user = await prisma.user.create({
+      data: {
+        password,
+        username,
+      },
+    });
+    return res
+      .cookie("user", JSON.stringify(user), {
+        maxAge: 3600000,
```

Comment:
Constants like this one here should be stored in const variables. Please store it in this file as `const cookieMaxAgeMiliseconds = 60 * 60 * 1000;`